### PR TITLE
Add user agent endpoint [ECR-803]

### PR DESCRIPTION
**Pull request by SerhiiOryshych**

### CODE REVIEW

File: exonum/src/api/public/system.rs [hunk 1]
```diff
@@ -52,30 +53,36 @@ impl SystemApi {
         }
     }
 
-    fn mempool_info(&self) -> MemPoolInfo {
-        MemPoolInfo { size: self.pool.read().expect("Expected read lock").len() }
+    fn mempool_info(self, router: &mut Router) {
+        let mempool = move |_: &mut Request| -> IronResult<Response> {
+            let info = MemPoolInfo { size: self.pool.read().expect("Expected read lock").len() };
+            self.ok_response(&::serde_json::to_value(info).unwrap())
```
Comment (vitvakatu) [inline exonum/src/api/public/system.rs]: @DarkEld3r how did you missed `::...` syntax here? :D It think it's better to import `to_value` into scope

File: exonum/src/api/public/system.rs [hunk 2]
```diff
@@ -52,30 +53,36 @@ impl SystemApi {
...
+        router.get("/v1/mempool", mempool, "mempool");
     }
 
-    fn healthcheck_info(&self) -> HealthCheckInfo {
-        HealthCheckInfo { connectivity: !self.shared_api_state.peers_info().is_empty() }
+    fn healthcheck_info(self, router: &mut Router) {
+        let healthcheck = move |_: &mut Request| -> IronResult<Response> {
+            let info =
+                HealthCheckInfo { connectivity: !self.shared_api_state.peers_info().is_empty() };
+            self.ok_response(&::serde_json::to_value(info).unwrap())
```
Comment (vitvakatu) [inline exonum/src/api/public/system.rs]: Same here

File: exonum/src/api/public/system.rs [hunk 3]
```diff
@@ -52,30 +53,36 @@ impl SystemApi {
...
+        };
+        router.get("/v1/healthcheck", healthcheck, "healthcheck");
+    }
+
+    fn user_agent_info(self, router: &mut Router) {
+        let user_agent = move |_: &mut Request| -> IronResult<Response> {
+            let info = user_agent::get();
+            self.ok_response(&::serde_json::to_value(info).unwrap())
```
Comment (vitvakatu) [inline exonum/src/api/public/system.rs]: aaand here

File: exonum/src/api/public/system.rs (hunk 1 above)
Comment (DarkEld3r) [reply exonum/src/api/public/system.rs]: As for me, `to_value` functions itself very non-descriptive. I prefer the following:
```rust
use serde_json;

self.ok_response(&serde_json::to_value(info).unwrap())
```

**Pull request merged**